V2/tezex/src/components/session/WalletConnected.tsx

```diff
@@ -0,0 +1,12 @@
+import { FC } from "react";
+import { useSession} from "../../hooks/session";
+
+export const WalletConnected: FC = (props) => {
```

Comment:
In order to properly use TypeScript, could we please properly type `props` everywhere?

This produces a `TS2339` for me.